packages/number-field/test/number-field.test.ts

```diff
@@ -462,11 +459,11 @@ describe('NumberField', () => {
                 buttonDownRect.x + buttonDownRect.width / 2,
                 buttonDownRect.y + buttonDownRect.height / 2,
             ];
```

Comment:
GH won't let me suggest all of these (too far from touched code), but wherever we're responsible for creating the original value and for consuming it, we should specify the type on the value itself instead of asserting it later:

```suggestion
            ] as [number, number];
```

Reply:
Good call. Added on the `const variable: [number, number]` version wherever used here.

Also, got CI to pass, so 🥳